src/models/GllobalValidations/DescriptionValidation.tsx

```diff
@@ -182,6 +181,35 @@ import{
     {
         required: true,
         message: emptyInput(),
-    }
-    
+    },
+    Inputs: 
+    [
+        {
+            pattern: /^\s*\S.*$/,
```

Comment:
Can there be two or more spaces between words?

Reply:
yes

Reply:
it only prevent entering just whitespaces, as was in requirements